client/gui/CIntObject.h

```diff
@@ -217,3 +215,13 @@ class WindowBase : public CIntObject
 protected:
 	void close();
 };
+
+class IStatusBar
+{
+public:
+	virtual ~IStatusBar();
+	virtual void clear() = 0;
+	virtual void clearMatching(const std::string & Text) = 0;

```

Comment:
If I look at this interface, it's completely unclear what clearMatching shall do